app/src/main/java/com/nwa/smartgym/lib/adapters/AddDeviceAdapter.java

```diff
@@ -0,0 +1,87 @@
+package com.nwa.smartgym.lib.adapters;
+
+import android.bluetooth.BluetoothAdapter;
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
+import android.widget.TextView;
+
+import com.nwa.smartgym.R;
+import com.nwa.smartgym.models.Device;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by robin on 29-5-16.
+ */
+public class AddDeviceAdapter extends ArrayAdapter<Device> {
+    private Map<String, Device> devicesToBePersisted;
```

Comment:
Nee, zoals de variable naam al aanduid moeten deze devices not persisted worden. Dus een UUID is niet mogelijk, dit is een MAC address